### DISCUSSION TITLE
GitHub Copilot being weird

### DISCUSSION THREAD

**Discussion by dave9123** [community/community#10751] [Copilot Conversations]
When I typed:
// Open youtube.com when the code is running.
It outputs `// Open youtube.com/watch?v=dQw4w9WgXcQ
// Open main.js
// Open main.js
// Open main.js
// Open main.js`
And it keep saying that while I pressed tab + enter

**Comment by davecheney** [comment]
If there is little context to go on Copilot can attracted too repetition. If you give it more context, ie add a comment, write more of the function yourself, it may offer a better solution.

**Reply by dave9123** [reply]
What should I type?

**Reply by davecheney** [reply]
You should type more to give Copilot more context to prepare a suggestion for you.

**Reply by dave9123** [reply]
Type // Open a website youtube.com?
